src/apps/pulse/hooks/useIntentSdk.ts

```diff
@@ -98,9 +146,62 @@ export default function useIntentSdk() {
     connectors,
   ]);
 
+  useEffect(() => {
+    if (
+      !areModulesInstalled &&
+      intentSdk &&
+      payingTokens &&
+      payingTokens.length > 0
+    ) {
+      const { chainId } = payingTokens[0];
+      setIsFetching(true);
+      intentSdk
+        .isWalletReadyForPulse(chainId)
+        .then((res) => {
+          if (res) {
+            // eslint-disable-next-line no-console
+            console.log('isWalletReadyForPulse: ', res);
+            setAreModulesInstalled(true);
+          } else {
+            setAreModulesInstalled(false);
+          }
+          setIsFetching(false);
+        })
+        .catch((err) => {
+          console.error('err:: ', err);
+          setIsFetching(false);
+          setAreModulesInstalled(false);
+        });
+    }
+  }, [intentSdk, payingTokens, areModulesInstalled]);
+
+  const installModules = async () => {
+    if (!payingTokens?.length || !intentSdk) return;
+    const { chainId } = payingTokens[0];
+    setIsInstalling(true);
+    try {
+      const res: Transactions[] = await intentSdk.enablePulseTrading(chainId);
+      const ok = await sendTransactions(res, chainId);
+      setAreModulesInstalled(ok);
+    } catch (err) {
+      console.error('Installation failed:: ', err);
```

Comment:
```suggestion
      console.error('Pulse trading - Installation failed: ', err);
```

Remember to make the messages relevant to everyone if you are leaving them in